Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.mm

```diff
@@ -137,6 +153,62 @@ - (void)dealloc
 
 @end
 
+@implementation _WKApplicationManifestShortcut
+
++ (BOOL)supportsSecureCoding
+{
+    return YES;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
+    if (!(self = [self init]))
+        return nil;
+
+    _name = [[coder decodeObjectOfClass:[NSString class] forKey:@"name"] copy];
+    _url = [[coder decodeObjectOfClass:[NSURL class] forKey:@"url"] copy];
+    _icons = [[coder decodeObjectOfClasses:[NSSet setWithArray:@[[NSArray class], [_WKApplicationManifestIcon class], [NSString class]]] forKey:@"icons"] copy];
+
```

Comment:
I would recommend using `RetainPtr` instead of manual `-retain` / `-release`

Reply:
Switched to using `RetainPtr`, thank you for the patch!